spaces/wolfdefi/index.json

```
@@ -0,0 +1,20 @@
+{
+  "key": "woldefi",
```

The key of your space must also be the folder name and skin class name. 
```suggestion
  "key": "wolfdefi",
```